### PR TITLE
Add possibility to retrive the number of tracked pageviews in JS tracker

### PR DESCRIPTION
**Pull request by tsteur**
Useful to detect eg if a page view has been already tracked or not.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ The Product Changelog at **[piwik.org/changelog](http://piwik.org/changelog)** l
 * A new event `Db.getActionReferenceColumnsByTable` has been added in case a plugin defines a custom log table which references data to the log_action table 
 * A new JavaScript tracker method `getPiwikUrl` has been added to retrieve the URL of where the Piwik instance is located
 * A new JavaScript tracker method `getCurrentUrl` has been added to retrieve the current URL of the website. 
+* A new JavaScript tracker method `getNumTrackedPageViews` has been added to retrieve the number of tracked page views.
```
Comment (sgiehl) [inline CHANGELOG.md]: Won't that description be a bit confusing? Someone might think it makes it possible to retrieve the number of pageviews tracked within the visit. But actually it only returns the number of `trackPageView` calls done within the current loaded page.

Comment (tsteur) [inline CHANGELOG.md]: Good point 👍

**Comment by tsteur** [comment]
@sgiehl I updated PR

**Comment by sgiehl** [comment]
Haven't had time to check out the branch and test it. But code looks good and as tests are passing I guess it'd be fine to merge.

**Comment by tsteur** [comment]
👍

**Pull request merged**